assembl/static2/js/app/components/home/header/statistic.jsx

```diff
@@ -39,49 +41,35 @@ class Statistic extends React.Component {
   render() {
     const { rootIdea, numParticipants, totalSentiments, visitsAnalytics } = this.props.data;
     const elementsProps = [
-      { iconName: 'sentiment-neutral', count: totalSentiments, translateValue: 'home.sentiments' },
-      { iconName: 'profil', count: numParticipants, translateValue: 'home.participant' }
+      { iconName: 'sentiment-neutral', metricValue: totalSentiments, metricNameTranslateKey: 'home.sentiments' },
+      { iconName: 'profil', metricValue: numParticipants, metricNameTranslateKey: 'home.participant' }
     ];
     if (rootIdea) {
       elementsProps.push({
         iconName: 'message',
-        count: rootIdea.numPosts,
-        translateValue: 'home.contribution'
+        metricValue: rootIdea.numPosts,
+        metricNameTranslateKey: 'home.contribution'
       });
     }
 
     if (visitsAnalytics) {
       if ('sumVisitsLength' in visitsAnalytics && visitsAnalytics.sumVisitsLength > 0) {
         const totalSeconds = visitsAnalytics.sumVisitsLength;
-        const totalSecondsToReadableDuration = function (seconds) {
-          const secondsAsNumber = parseInt(seconds, 10); // don't forget the second param
-          let hours = Math.floor(secondsAsNumber / 3600);
-          let minutes = Math.floor((secondsAsNumber - hours * 3600) / 60);
-          let secondsRemaining = secondsAsNumber - hours * 3600 - minutes * 60;
-
-          if (hours < 10) {
-            hours = `0${hours}`;
-          }
-          if (minutes < 10) {
-            minutes = `0${minutes}`;
-          }
-          if (seconds < 10) {
-            secondsRemaining = `0${secondsRemaining}`;
-          }
-          return `${hours}h ${minutes}m ${secondsRemaining}s`;
-        };
-        const readableDuration = totalSecondsToReadableDuration(totalSeconds);
+        const secondsAsNumber = parseInt(totalSeconds, 10); // don't forget the second param
```

Comment:
why do we need the second param here? According to https://www.w3schools.com/jsref/jsref_parseInt.asp if the string does not starts with `0x` or `0` we don't need it

Reply:
eslint gives you an error if you don't specify the radix

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt clearly says:
> Always specify this parameter to eliminate reader confusion and to guarantee predictable behavior. Different implementations produce different results when a radix is not specified, usually defaulting the value to 10.

Reply:
Ok, good. I thought I missed something when I saw the comment